theano/gpuarray/tests/config.py

```diff
@@ -17,7 +17,7 @@
 
 if theano.config.mode == 'FAST_COMPILE':
     mode_with_gpu = theano.compile.mode.get_mode('FAST_RUN').including('gpuarray').excluding('gpu')
-    mode_without_gpu = theano.compile.mode.get_mode('FAST_RUN').excluding('gpuarray')
+    mode_without_gpu = theano.compile.mode.get_mode('FAST_RUN').excluding('gpuarray', 'gpu')
 else:
     mode_with_gpu = theano.compile.mode.get_default_mode().including('gpuarray').excluding('gpu')
-    mode_without_gpu = theano.compile.mode.get_default_mode().excluding('gpuarray')
+    mode_without_gpu = theano.compile.mode.get_default_mode().excluding('gpuarray', 'gpu')
```

Comment:
It *disables* the old backend, I think. Maybe this is only a problem on my system (I made it a separate commit).

If I don't also exclude `'gpu'`, the tests that check for a CPU reference implementation of the pooling ops fail. For example, the CPU `Pool` gets replaced by `theano.sandbox.cuda.dnn.GpuDnnPool`. (In [test_pooling](https://github.com/Theano/Theano/blob/166cac0/theano/gpuarray/tests/test_dnn.py#L271-L276), for instance.)